patches/chrome-BUILD.gn.patch

```diff
@@ -42,41 +44,51 @@ index fe09f99a91e3b1d2abfb335ec2771d1ed8592d87..e2ddd668d9c9a30708a348b94f7d1bdd
            "-v",
            rebase_path(chrome_version_file, root_build_dir),
            "-g",
-@@ -670,7 +680,7 @@ if (is_win) {
-       # framework itself, that would cause a cyclical dependency. Instead,
+@@ -671,6 +677,7 @@ if (is_win) {
        # this dependency directly copies the file into the framework's
        # resources directory.
--      public_deps += [ ":chrome_framework_widevine_signature" ]
-+      #public_deps += [ ":chrome_framework_widevine_signature" ]
+       public_deps += [ ":chrome_framework_widevine_signature" ]
++      public_deps -= [ ":chrome_framework_widevine_signature" ]
      }
    }
  
-@@ -1115,6 +1125,10 @@ if (is_win) {
+@@ -707,6 +714,7 @@ if (is_win) {
+       "--scm=0",
+     ]
+   }
++  copy("brave_helper_plist") { deps = [ "//brave:brave_helper_plist" ] sources = [ "$root_gen_dir/brave/brave_helper_plist_tweaked.plist" ] outputs = [ "$root_gen_dir/chrome/{{source_file_part}}" ] }
+ 
+   compile_entitlements("entitlements") {
+     entitlements_templates = [ "app/app-entitlements.plist" ]
+@@ -725,7 +733,7 @@ if (is_win) {
+ 
+       output_name = chrome_helper_name + invoker.helper_name_suffix
+ 
+-      info_plist_target = ":chrome_helper_plist"
++      info_plist_target = ":brave_helper_plist"
+       extra_substitutions = [
+         "CHROMIUM_BUNDLE_ID=$chrome_mac_bundle_id",
+         "CHROMIUM_SHORT_NAME=$chrome_product_short_name",
+@@ -1115,6 +1123,10 @@ if (is_win) {
      if (is_chrome_branded) {
        deps += [ ":default_apps" ]
      }
 +    if (brave_chromium_build) {
```

Comment:
I think we can delete `if (brave_chromium_build)`. it doesn't have much meaning here.

Reply:
yea, but this patch also has some other issues and I didn't want this to turn into a complete overhaul of it. I already did a little too much by removing https://github.com/brave/brave-core/pull/5064/files/7909d7aac014308f769e4620eb11383771af8872#diff-592bd965aabfe4ab456e5454131e530eL77 which really should have been a separate PR

Reply:
same with https://github.com/brave/brave-core/pull/5064/files/7909d7aac014308f769e4620eb11383771af8872#diff-592bd965aabfe4ab456e5454131e530eR51
I had actually meant to split them out, but then forgot about it

Reply:
I see - let's cleanup as a f/u :)